<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindowR</class>
 <widget class="QMainWindow" name="MainWindowR">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="dockOptions">
   <set>QMainWindow::AllowTabbedDocks|QMainWindow::AnimatedDocks</set>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>150</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>department</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_patientsHistory">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>150</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>patient history</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_doctorSchedule">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>150</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>doc schedule</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_6">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>319</y>
      <width>160</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>beds</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_dischargeSlip">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>319</y>
      <width>160</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>discharge slip</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_birthTroll">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>319</y>
      <width>160</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>birth troll</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_deathTroll">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>479</y>
      <width>160</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>death troll</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_bill">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>479</y>
      <width>160</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>bill</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_appointmentForm">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>479</y>
      <width>160</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>appointment form</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_department">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>40</y>
      <width>141</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_patientHistory">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>40</y>
      <width>151</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_beds">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>210</y>
      <width>151</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_dischargeSlip">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>210</y>
      <width>151</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_doctorSchedule">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>40</y>
      <width>151</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_birthTroll">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>210</y>
      <width>151</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_deathTroll">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>370</y>
      <width>151</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_bill">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>370</y>
      <width>151</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_appointmentForm">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>370</y>
      <width>151</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="separator"/>
  </widget>
  <action name="actionHome">
   <property name="text">
    <string>Home</string>
   </property>
  </action>
  <action name="actionDepartments">
   <property name="text">
    <string>Departments</string>
   </property>
  </action>
  <action name="actionPatients_History">
   <property name="text">
    <string>Patient's History</string>
   </property>
  </action>
  <action name="actionDoctor_s_Schedule">
   <property name="text">
    <string>Doctor's Schedule</string>
   </property>
  </action>
  <action name="actionBeds">
   <property name="text">
    <string>Beds</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
